101-basics/variables/README.md

```diff
@@ -1,6 +1,30 @@
 # Variables
 
-Go dilinde üç farklı şekilde değişken tanımı yapılabilir.
+Go dilinde oldukça farklı şekillerde değişken tanımı yapılabilir.
+
+```go
+var a int // sadece tanımlama
+var a2 int = 1 // tanımlama ve değer atama
```

Comment:
Burada tipi belirtmek opsiyoneldir. Belirtmediğimiz durumda da atanan değerin tipi şeklinde tanımlanacaktır. Buna ek olarak tüm yorum satırlarını ilgili bloğun üstünde, bloğu açıklayan cinste kullanmamız daha doğru olabilir. Örneğin: //Bu kulanım değişkene bir değer atayarak tanımlanın bir örneğidir.

Reply:
Tip belirtmenin opsiyonel olduğunun anlaşılması için bu satırların altında şu tanımlama örneğini de vermiştim:
```go
var b = "go turkiye"
```

Tamamdır, yorum satırlarını daha anlaşılır bir hale getireceğim.